source/addComment.js

```diff
@@ -1,3 +1,19 @@
 $(document).ready(function(){
-  // add your code here
+  $('#new_comment_button').on("click", function()  {
+    $('body').append('<form id="new-comment"></form>');
+    $('#new-comment').append('<label for "comment-content">Comment</label>')
```

Comment:
You can append multiple line of HTML - you'd just indent/nest as you'd see in an HTML document! That way you only need to grab $('#new-comment') and call append() once.